docs/src/architectures/pet.rst

```diff
@@ -161,6 +161,98 @@ result in a model that is about 1.5 times more lightweight and faster, with an
 expected minimal deterioration in accuracy.
 
 
+Description of Hyperparameters
+------------------------------
+
+``RANDOM_SEED``: random seed
+``CUDA_DETERMINISTIC``: if applying PyTorch reproducibility settings
+``MULTI_GPU``: use multi-GPU training (on one node) using DataParallel from 
+PyTorch-Geometric
+
+``MODEL_TO_START_WITH``: If not None, then training will be continued from this 
+checkpoint (only relates to weights of the model, and not to the state of the 
+learning rate scheduler and optimizer)
+
+``R_CUT``: cutoff radius
+``CUTOFF_DELTA``: width of the transition region for a cutoff function used by 
+PET to ensure smoothness with respect to the (dis)appearance of atoms at the 
+cutoff sphere
+
+``GLOBAL_AUG``: whether to use global augmentation or a local one, rotating 
+atomic environments independently
+
+``USE_ENERGIES``: whether to use energies for training
+``USE_FORCES``: whether to use forces for training
+``SLIDING_FACTOR``: sliding factor for exponential sliding averages of MSE in 
+energies and forces in our combined loss definition
+``ENERGY_WEIGHT``: $w_{E}$, dimensionless energy weight in our combined loss 
+definition
+
+``N_GNN_LAYERS``: number of message-passing blocks
+``TRANSFORMER_D_MODEL``: was denoted as d_{pet} in the main text
```

Comment:
I don't see a d_pet above?

Reply:
It is this one:

``TRANSFORMER_D_MODEL``: was denoted as d_{pet} in the main text

Reply:
No, I mean this line is referring to `d_pet`, but `d_pet` can't be found anywhere else in this page

Reply:
I referred to the paper. I have a added a clarification.